webui/react/src/pages/Wait.tsx

```diff
@@ -17,7 +17,7 @@ import { WaitStatus } from 'wait';
 
 import css from './Wait.module.scss';
 
-interface Params {
+type Params = {
```

Comment:
ditto: conditional params? unless the router is preventing this page from rendering if the params are not provided?

Reply:
i use `assertIsDefined` to check if `undefined` or not. Even if the variable is undefined, the page will redirect to default page, and `assertIsDefined` doesnt actually throw error